src/algebra/group_power/lemmas.lean

```diff
@@ -83,6 +83,15 @@ begin
   exact is_unit_of_invertible x
 end
 
+lemma pow_smul [monoid A] [mul_action M A] [is_scalar_tower M A A] [smul_comm_class M A A]
```

Comment:
Should this be
```suggestion
lemma smul_pow [monoid A] [mul_action M A] [is_scalar_tower M A A] [smul_comm_class M A A]
```
Since the smul is the right argument to pow?

Reply:
We have a
```
@[simp] lemma smul_pow (x : α) (m : β) (n : ℕ) :
  x • m ^ n = (x • m) ^ n :=
```
added by you recently. What does the naming convention say here?

Reply:
That existing lemma should be primed to match smul_mul'.

It probably also should be relaxed to mul_distrib_mul_action, and moved to this file